### PR TITLE
Feature/update angular 9

### PR DESCRIPTION
**Pull request by Lukasz-Bialkowski**

### CODE REVIEW

File: tsconfig.json [hunk 1]
```diff
@@ -1,5 +1,8 @@
 {
   "compileOnSave": false,
+  "angularCompilerOptions": {
+    "enableIvy": false
```
Comment (AntoineXevlabs) [inline tsconfig.json:4]: Ivy might be a good solution

File: package.json [hunk 2]
```diff
@@ -13,33 +13,33 @@
...
-    "@angular/compiler": "^8.1.2",
-    "@angular/core": "^8.1.2",
-    "@angular/forms": "^8.1.2",
-    "@angular/material": "^8.1.1",
-    "@angular/platform-browser": "^8.1.2",
-    "@angular/platform-browser-dynamic": "^8.1.2",
-    "@angular/router": "^8.1.2",
+    "@angular/animations": "9.0.0",
+    "@angular/cdk": "^9.2.0",
+    "@angular/common": "9.0.0",
```
Comment (AntoineXevlabs) [inline package.json:18]: I think latest stable version is 9.1.9 we might go for this one

File: tsconfig.json (hunk 1 above)
Comment (Lukasz-Bialkowski) [reply tsconfig.json:4]: Ivy was not well supported in NX and was discouraged. How does it look like now?
With Ivy enabled we have a big warn in a console that as a library Ivy should be disabled.

Comment (Lukasz-Bialkowski) [reply tsconfig.json:4]: Screenshot 2020-06-04 at 09 33 42

Comment (mthomasson) [reply tsconfig.json:4]: https://github.com/nrwl/nx/issues/1364
I thnk we could try it now!
But maybe not in this lib, let's try it in an NX project first, to make sure everything is ok

**Pull request merged**